src/components/Button/index.tsx

```diff
@@ -318,6 +279,7 @@ function Button({
                 isDisabled && !danger && !success ? styles.buttonDisabled : undefined,
                 shouldRemoveRightBorderRadius ? styles.noRightBorderRadius : undefined,
                 shouldRemoveLeftBorderRadius ? styles.noLeftBorderRadius : undefined,
+                // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
```

Comment:
Why not nullish coalescing in this place?

Reply:
`icon || shouldShowRightIcon ? styles.alignItemsStretch : undefined`
i gues for if statments we should use normal OR operator

Reply:
You can try `!!icon || shouldShowRightIcon....`, eslint should pass it so you can remove eslint disable